Gemfile.jruby-1.9.lock.defaults

```diff
@@ -241,7 +238,7 @@ GEM
     logstash-input-exec (0.1.3)
       logstash-codec-plain
       logstash-core (>= 1.4.0, < 2.0.0)
-    logstash-input-file (0.1.6)
+    logstash-input-file (0.1.7)
       addressable
       filewatch (= 0.6.1)
```

Comment:
Good catch
The PR was created after the release of the filewatch.
``
filewatch = 0.6.2
logstash-input-file  = 0.1.8
``